source/slang/slang-ir-early-raytracing-intrinsic-simplification.cpp

```diff
@@ -132,11 +132,12 @@ namespace Slang
                 {
                     auto op = i->getOperand(0);
                     IRInst* globalVar = cache->getRayVariableFromLocation(op, i->getOp());
-                    auto builder = IRBuilder(i);
-                    builder.setInsertBefore(i);
+                    auto builder = IRBuilder(i->getParent());
+                    builder.setInsertInto(i->getParent());
```

Comment:
You want to insert before `i`, right?,  The code after this change will insert spirvASM at the end of the parent.